packages/runtime-common/realm.ts

```diff
@@ -1696,6 +1703,71 @@ export class Realm {
     });
   }
 
+  private transformResultsToPrerenderedCardsDoc(results: {
+    prerenderedCards: PrerenderedCard[];
+    meta: QueryResultsMeta;
+  }) {
+    let { prerenderedCards, meta } = results;
+    let uniqueCssModules = new Map(); // every instance in fetched results will have css (if it exists) - to avoid duplication in the response, we'll store unique css modules here
```

Comment:
instead of determining css uniqueness at this layer, perhaps it would be more performant to do that as part of the DB query. Like the DB query could return something like:
```
{
  prerenderedCard: {
    html: string;
    cssModule: string
  }[];
  cssModules: {
    css: string;
    module: string;
  }[]
  }